packages/core/components/SearchBox/test/SearchBox.test.tsx

```diff
@@ -4,19 +4,15 @@ import { noop } from "lodash";
 import * as React from "react";
 import sinon from "sinon";
 
-import SearchBoxForm from "..";
 import FileFilter from "../../../entity/FileFilter";
 
```

Comment:
This is a convention special for test files for making it especially clear which file is being tested.I think it is because file name and component names can change so this is the most obvious spot to see that.